public/docs/_examples/cli-quickstart/ts/src/app/app.module.ts

```diff
@@ -0,0 +1,20 @@
+import { BrowserModule } from '@angular/platform-browser';
```

Comment:
Personally I don't like aligning import sentences, is a bit weird sometimes. But that is just me.